source/help/getting-started/signing-in.rst

```diff
@@ -0,0 +1,125 @@
+Signing in
```

Comment:
Please also update the index file due to the .md > .rst conversion: https://github.com/mattermost/docs/blob/master/source/guides/user.rst